locale/es/get-involved/index.md

```diff
@@ -7,22 +7,21 @@ layout: contribute.hbs
 
 ## Discusión de la comunidad
 
-* La [lista de errores en](https://github.com/nodejs/node/issues) es el lugar para discutir las características del núcleo de Node.js.
+* La [lista de errores](https://github.com/nodejs/node/issues) es el lugar para discutir las características del núcleo de Node.js.
 * Para chatear en tiempo real sobre el desarrollo de Node vaya a `irc.freenode.net` en el canal `#node.js` usando un [cliente de IRC](https://es.wikipedia.org/wiki/Anexo:Clientes_IRC) ó conéctese con su navegador al canal usando [WebChat de freenode](https://webchat.freenode.net/#node.js).
```

Comment:
Again, another 'o' 🙃 
[Never graphic accent in 'o' as a conjunction](https://www.rae.es/consultas/la-conjuncion-o-siempre-sin-tilde-incluso-entre-cifras)

```suggestion
* Para chatear en tiempo real sobre el desarrollo de Node vaya a `irc.freenode.net` en el canal `#node.js` usando un [cliente de IRC](https://es.wikipedia.org/wiki/Anexo:Clientes_IRC) o conéctese con su navegador al canal usando [WebChat de freenode](https://webchat.freenode.net/#node.js).
```

Reply:
Entendido, Gracias.

I did not know that conjunction 💯